manifests/server/config.pp

```diff
@@ -210,14 +211,20 @@
     }
 
     file { $dbpath:
-      ensure   => directory,
-      mode     => '0755',
-      owner    => $user,
-      group    => $group,
-      recurse  => true,
-      purge    => false,
-      checksum => 'none',
-      require  => File[$config],
+      ensure  => directory,
+      mode    => '0755',
+      owner   => $user,
+      group   => $group,
+      require => File[$config],
+    }
+
+    if $dbpath_fix {
+      exec { 'fix dbpath permissions':
+        command     => "chown -R ${user}:${group} ${dbpath}",
+        path        => ['/usr/bin', '/bin'],
+        refreshonly => true,
```

Comment:
So this would never continually manage the permissions. You could achieve this by doing something like...
````````
  exec { 'fix dbpath permissions':
    command   => "chown -R ${user}:${group} ${dbpath}",
    path      => ['/bin/','/usr/bin']
    onlyif    => "find ${dbpath} -not -user ${user} -o -not -group ${group} -print -quit | grep -q '.*'",
    subscribe => File[$dbpath]
  }
```